src/cmd_line/subparser.ts

```diff
@@ -99,6 +100,11 @@ export const commandParsers = {
 
   reg: parseRegisterCommandArgs,
 
+  dig: parseDigraphCommandArgs,
+  digr: parseDigraphCommandArgs,
+  digraph: parseDigraphCommandArgs,
+  digraphs: parseDigraphCommandArgs,
+
```

Comment:
I'm not familiar with the vim digraph command. Are all these commands synonymous?

Reply:
Yeah, they all map to the same two commands: https://github.com/vim/vim/blob/master/runtime/doc/digraph.txt#L27-L39